build/docker-entrypoint.sh

```diff
@@ -1,19 +1,27 @@
 #!/usr/bin/env bash
 
-if [ $1 == "dst-server" ]; then
-	set -e
+# Exit immediately on non-zero return codes.
+set -e
```

Comment:
Moved this to the top, outside of the `if` condition block so it will be set for every command run on the container. Makes sense?